channels/views/moderators_test.py

```diff
@@ -22,6 +22,50 @@ def test_list_moderators(client, private_channel_and_contributor, staff_user):
     assert resp.json() == [{'moderator_name': staff_user.username}]
 
 
+def test_list_moderators_staff(  # pylint: disable=too-many-arguments
+        client, private_channel, staff_user, staff_api, reddit_factories, staff_jwt_header
+):
+    """
+    List moderators in a channel
```

Comment:
Minor nitpick: Change to something like `List moderators in a channel as a staff user`?